code/modules/mob/living/living_say.dm

```diff
@@ -451,6 +451,11 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 	return message
 
 /mob/living/proc/radio(message, list/message_mods = list(), list/spans, language)
+	//SKYRAT EDIT ADDITION BEGIN
+	if((message_mods[MODE_HEADSET] || message_mods[RADIO_EXTENSION]) && !(mobility_flags & MOBILITY_USE)) // If can't use items, you can't press the button
+		to_chat(src, span_warning("You can't use the radio right now!"))
```

Comment:
maybe have the message give a little more specificity why? ("you can't reach the button!" or smth)

Reply:
done